detekt/detekt-config.yml

```diff
@@ -114,7 +114,7 @@ complexity:
   LongParameterList:
     active: true
     functionThreshold: 6
-    constructorThreshold: 7
+    constructorThreshold: 10
```

Comment:
Is this something we can keep at the default? I think this rule might be good to keep. Could we refactor where this rule was failing?